deps.edn

```diff
@@ -0,0 +1,11 @@
+{:paths ["src"]
```

Comment:
it is quite confusing to have two sources of dependencies, because people would need to keep them both up to date

Reply:
I forgot about this PR as well, this was added to do some troubleshooting. I'll look into fixing up this PR again next weekend.